right-for-life_front/src/components/Footer/Footer.jsx

```diff
@@ -1,46 +1,60 @@
 import React from "react";
 import { NavLink } from "react-router-dom";
 
+import { withFetchDataIndicators } from "../../hoc/withFetchDataIndicators";
+import { API } from "../../rootConstants";
 import { DonateButton } from '../DonateButton';
 
-import logo from "../../assets/logo.jpg";
-
-export const Footer = () => {
+const Footer = ({ data }) => {
+	const { instagram, facebook, phone, email} = data;
 	const iconsClassName = 'mx-2 text-3xl';
 
 	return (
-		<footer className="flex flex-shrink-0 justify-center items-center py-3 bg-white shadow-md font-bold text-xl text-gray-600 mb-16 lg:mb-0">
-			<ul className="flex">
+		<footer className="flex flex-col lg:flex-row xl:flex-row flex-shrink-0 justify-center items-center py-3 bg-white shadow-md font-bold text-xl text-gray-600 my-16 lg:mb-0">
+			<ul className="flex my-2">
+				<li>
+					<a
+						href={instagram}
+						target="_blank"
+						rel='noopener noreferrer'
+					>
+						<i className={`fab fa-instagram-square text-pink-500 ${iconsClassName}`}></i>
+					</a>
+				</li>
 				<li>
 					<a
-						href="https://www.facebook.com/FOND.Mi.za.pravo.na.zhizn/?ref=page_internal"
+						href={facebook}
 						target="_blank"
 						rel='noopener noreferrer'
 					>
 						<i className={`fab fa-facebook-square text-blue-700 ${iconsClassName}`}></i>
 					</a>
 				</li>
 				<li>
-					<a href="tel:093235037">
+					<a href={`tel:${phone.replace('\'','')}`}>
 						<i className={`fas fa-phone-square-alt text-green-500 ${iconsClassName}`}></i>
 					</a>
 				</li>
 				<li>
-					<a href="mailto:olyaum76@gmail.com">
+					<a href={`mailto:${email}`}>
 						<i className={`fas fa-envelope text-red-500 ${iconsClassName}`}></i>
 					</a>
 				</li>
 			</ul>
-			<span className="mx-24">Designed by SoftServe Inc.</span>
+			<span className="hidden lg:flex xl:flex text-center mx-24 my-2">Designed by SoftServe Inc.</span>
 			<NavLink
-				className="mx-4 cursor-pointer text-gray-600 hover:text-gray-700"
+				className="hidden lg:flex xl:flex mx-4 cursor-pointer text-gray-600 hover:text-gray-700"
```

Comment:
`lg:flex xl:flex`
You don’t need to duplicate the style to a higher screens, you can just `write lg:flex`